app/main.py

```diff
@@ -152,32 +162,37 @@ async def delete(request):
     ids = post.get('ids')
     where = post.get('where')
     if not ids or where not in ['queue', 'done']:
+        log.error("Bad request: missing 'ids' or incorrect 'where' value")
         raise web.HTTPBadRequest()
     status = await (dqueue.cancel(ids) if where == 'queue' else dqueue.clear(ids))
+    log.info(f"Download delete request processed for ids: {ids}, where: {where}")
     return web.Response(text=serializer.encode(status))
 
 @routes.post(config.URL_PREFIX + 'start')
 async def start(request):
     post = await request.json()
     ids = post.get('ids')
+    log.info(f"Received request to start pending downloads for ids: {ids}")
     status = await dqueue.start_pending(ids)
     return web.Response(text=serializer.encode(status))
 
 @routes.get(config.URL_PREFIX + 'history')
 async def history(request):
     history = { 'done': [], 'queue': [], 'pending': []}
 
-    for _ ,v in dqueue.queue.saved_items():
+    for _, v in dqueue.queue.saved_items():
         history['queue'].append(v)
-    for _ ,v in dqueue.done.saved_items():
+    for _, v in dqueue.done.saved_items():
         history['done'].append(v)
     for _ ,v in dqueue.pending.saved_items():

```

Comment:
I see that the same whitespace mishap exists here as above, perhaps worth fixing too while you're at it